mix.exs

```diff
@@ -96,7 +96,8 @@ defmodule Backpex.MixProject do
     [
       "guides/introduction/installation.md",
       "guides/introduction/translations.md",
-      "guides/advanced/full_text_search.md"
+      "guides/advanced/full_text_search.md",
+      "guides/upgrading/v0.2.md"
```

Comment:
There should also be added a "group for extras" below, I think.

```
"Upgrade Guides": ~r{guides/upgrading/.*}
```

Reply:
Done ✅